.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Team Lead
+* @junan
+
+# Team Members
+* @carryall @hoangmirs @Lahphim @malparty
```

Comment:
@olivierobert 
I've just found that this line will override the above line, so Junan won't be the code owner of this project.
This is the preview when I open a PR on react-templates:
![__53__Add_react-router_to_the_template_by_hoangmirs_·_Pull_Request__61_·_nimblehq_react-templates_🔊](https://user-images.githubusercontent.com/7344405/161518518-b01273e1-f073-4358-95be-a851cec1176e.png)
And it did not include Pae.

https://github.community/t/order-of-owners-in-codeowners-file/143073

How about adding the team lead at the beginning of each list instead?

Reply:
Nice catch!! 💡

Reply:
I agree with Hoand recommendation. To keep it readable, we might want to comment this single line with:
`#  Team Lead / Team Members`

Reply:
Oh, I thought they would be compounded rules 😭  So yes, we would need to use some kind of comments as suggested by Xavier but I would prefer to be more detailed:

```
# Team
# @junan is the Team Lead and the others are team members
* @junan @carryall @hoangmirs @Lahphim @malparty

# Engineering Leads
CODEOWNERS @nimblehq/engineering-leads
```

Reply:
@olivierobert Do we need to add the team lead to review `CODEOWNERS` file? As he/she might be the one who merges the PR 🤔 
```
# Team
# @junan is the Team Lead and the others are team members
* @junan @carryall @hoangmirs @Lahphim @malparty

# Engineering Leads
CODEOWNERS @nimblehq/engineering-leads @junan
```

Reply:
Actually, no need, as the previous rules will ensure that the whole team is added as reviewers.

Reply:
@olivierobert @hoangmirs Confirmed that we don't need to add TL at `CODEOWNERS` pattern. Time to make updates on all repositories again. 😢 

![image](https://user-images.githubusercontent.com/16315358/162140815-67af30d2-cc17-4db8-a7ef-2f83ac7ae293.png)

Reply:
Indeed @luongvo, all template repositories need to be updated again 😭 The issue was not found on [the initial PR](https://github.com/nimblehq/rails-templates/pull/328) to add the CODEOWNER file

Reply:
I fixed in https://github.com/nimblehq/gin-templates/pull/75/commits/f1d03851d1208f6dfa2934de291cf67c17c51880